www/nginx/pkg-descr

```diff
@@ -8,6 +8,9 @@ reuse, SSL offload and HTTP media streaming.
 Plugin Changelog
 ================
 
+1.17.1
+
+* add support for multiple http and https listen ports (contributed by Szeraax)
```

Comment:
Please aim for 1.18 and add a missing new line here

Reply:
√